sdk/core/azure-core/HISTORY.md

```diff
@@ -2,6 +2,8 @@
 # Release History
 
 -------------------
+## Version 1.1.2 Unreleased
```

Comment:
Did you do these version updates manually or use the scripts? Does the script add the "Unreleased" keyword to the heading?

Reply:
Do we have docs that instruct folks how to update versions with the scripts?

Reply:
Change log is updated manually. I have an issue pending with me to include change log update also in versioning tool. I will update the documentation on how to use the tool once this change to update change log is also completed